pyftpdlib/handlers.py

```diff
@@ -2992,12 +2992,14 @@ def ftp_OPTS(self, line):
             if line.count(' ') > 1:
                 raise ValueError('Invalid number of arguments')
             if ' ' in line:
-                cmd, arg = line.split(' ')
-                if ';' not in arg:
-                    raise ValueError('Invalid argument')
+                cmd, arg = line.split(' ') 
             else:
                 cmd, arg = line, ''
-            # actually the only command able to accept options is MLST
+            if cmd.upper() in ("UTF8", "UTF-8"):
+                self.respond('200 Always in UTF8 mode.')
+                return
+            if arg and ';' not in arg:
+                    raise ValueError('Invalid argument') 
```

Comment:
I wonder if this bit should be moved down to just above the `facts = [x.lower() for x in arg.split(';')]` line?
https://www.rfc-editor.org/rfc/rfc2389#section-4 says `command-options  = <format specified by individual FTP command>` and I think the semicolon in command-options is specific to MLST?

Reply:
Indentation of `raise ValueError` seems incorrect